tests/fetch/cached_xhr.cpp

```diff
@@ -0,0 +1,56 @@
+#include <string.h>
```

Comment:
This particular file, unlike the others, is used as a test in the suite, so didn't have the prefix `example_` there. The ones with `example_` are intended to be compact and unlike the tests they don't verify that the implementation is correct (those lines bloat up the tests by quite a bit).